test/Microsoft.AspNetCore.AzureAppServicesIntegration.Tests/HostingStartupTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Azure.KeyVault;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Microsoft.AspNetCore.AzureKeyVault.HostingStartup.Tests
+{
+    public class HostinStartupTests
+    {
+        [Fact]
+        public void Configure_AddsDataProtection()
+        {
+            Environment.SetEnvironmentVariable("ASPNETCORE_HostingStartup__KeyVault__DataProtection", null);
```

Comment:
HostingStartup doesn't seem important in these values.
`ASPNETCORE__KeyVault__DataProtection`
`ASPNETCORE__KeyVault__DataProtectionKey`
?

Reply:
Didn't want to take nice names and wanted it to show that it's only hosting startup specific.